scripts/speech_recognition/oomptimizer.py

```diff
@@ -0,0 +1,458 @@
+#!/usr/bin/env python
+import importlib
+import math
+import sys
+from numbers import Number
+
+import click
+import pytorch_lightning as pl
+import torch
+from lhotse import compute_num_samples
+from omegaconf import OmegaConf
+
+from nemo.core.classes.common import Model
+from nemo.core.neural_types import AudioSignal, LabelsType, LengthsType, NeuralType
```

Comment:
MaskType may also need to be supported at some point, as an alternative to LengthsType. I don't think it's a big deal, though.